content/tutorials/portenta-h7/vs-openmv-ml/content.md

```diff
@@ -0,0 +1,172 @@
+# Training a Machine Learning Model With Edge Impulse
+## Overview 
+
+This tutorial teaches you how to train a custom machine learning model using Edge Impulse and to build a simple object classifier. The Machine Learning (ML) model will use the TensorFlow Lite format and the classification example will run on OpenMV.
+
+### You Will Learn
+
+- To create datasets to be used for classification
+- How to train a ML model in Edge Impulse
+- To use OpenMV to run a classification example
+
+### Required Hardware and Software
+
+- [Portenta H7 board](https://store.arduino.cc/portenta-h7)
+- [Portenta Vision Shield - LoRa](https://store.arduino.cc/portenta-vision-shield-lora) or [Portenta Vision Shield - Ethernet](https://store.arduino.cc/usa/portenta-vision-shield)
+- Arduino [desktop IDE](https://www.arduino.cc/en/main/software) or Arduino [Web Editor](https://create.arduino.cc/)
```

Comment:
Do we need to mention the Arduino IDE here if we don't use it in the tutorial? Or can it be replaced with something like [OpenMV IDE](https://openmv.io/pages/download) ?

Reply:
Ha! Good catch